src/main/java/net/dv8tion/jda/api/entities/GuildChannel.java

```diff
@@ -179,6 +179,15 @@
     @Nonnull
     List<PermissionOverride> getRolePermissionOverrides();
 
+    /**
+     * Returns whether or not this GuildChannel is synced.
+     * <br>A GuildChannel is synced when its {@link net.dv8tion.jda.api.entities.PermissionOverride PermissionOverrides} match
+     * those of {@link #getParent() its parent category}. If the GuildChannel doesn't have a parent category this will return true.
+     * 
+     * @return Whether this GuildChannel is synced with its parent category.
+     */
+    boolean isSynced();
+
```

Comment:
```suggestion
    /**
     * Whether or not this GuildChannel's {@link net.dv8tion.jda.api.entities.PermissionOverride PermissionOverrides} match
     * those of {@link #getParent() its parent category}. If the channel doesn't have a parent category this will return true.
     * 
     * @return A boolean representing this condition.
     */
    boolean isSynced();

```